docs/en/observability/monitor-gcp.asciidoc

```diff
@@ -0,0 +1,474 @@
+:tutorial: tutorial
+:gcp: GCP
+
+[[monitor-gcp]]
+== Monitor Google Cloud Platform
+
+In this tutorial, you'll learn how to monitor your Google Cloud Platform ({gcp})
+deployments using Elastic Observability: Logs and Metrics.
+
+[discrete]
+=== What you'll learn
+
+You'll learn how to:
+
+- Set up a {gcp} Service Account
+- Ingest metrics using the {metricbeat-ref}/metricbeat-module-gcp.html[{metricbeat}
+Google Cloud Platform module] and view those metrics in {kib}.
+- Export GCP audit logs through Pub/Sub topics
+- Ingest logs using the {filebeat-ref}/filebeat-module-gcp.html[{filebeat}
+Google Cloud module] and view those logs in {kib}.
```

Comment:
Two of these don't have periods and two do–can you add or remove periods to make these consistent please

Reply:
This would be my recommendation, but either way works

```suggestion
- Set up a {gcp} Service Account.
- Ingest metrics using the {metricbeat-ref}/metricbeat-module-gcp.html[{metricbeat}
Google Cloud Platform module] and view those metrics in {kib}.
- Export GCP audit logs through Pub/Sub topics.
- Ingest logs using the {filebeat-ref}/filebeat-module-gcp.html[{filebeat}
Google Cloud module] and view those logs in {kib}.
```